tests/diagnostics/charged_particle_radiography/test_synthetic_radiography.py

```diff
@@ -542,9 +542,6 @@ def create_tracker_obj(**kwargs) -> cpr.Tracker:
     return sim
 
 
-tracker_obj_simulated = create_tracker_obj(field_weighting="nearest neighbor").run()
```

Comment:
After the changes from `autotyping`, `mypy` found that the `run` method here was returning `None`, which ended up being the value passed to `tracker_obj_simulated`.  This was one example of how static type checking can be helpful!